azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachines.java

```diff
@@ -32,20 +32,73 @@
     PagedList<VirtualMachineSize> listSizes(String region) throws CloudException, IOException;
```

Comment:
as discussed, since the two methods show up under different types, so even naming them the same seems ok.
But we can also name the one under VM as .availableSizes() and the one under VMs as .availableSizesByRegion()